### DISCUSSION TITLE
Error: Call to undefined method Joomla

### DISCUSSION THREAD

**Discussion by z4pe** [templaza/astroid-framework#644] [Q&A]
Hi Guys,

I am trying to update the astroid framework (2.4.1) of my joomla (3.10.12) site on wamp server before I update my live server, but when I install the 3.0.3 version or above, an internal server error message appears on the extension installation page. The framework has been updated, but I get this:  Error: Call to undefined method Joomla\CMS\Document\HtmlDocument::getWebAssetManager(): Call to undefined method Joomla\CMS\Document\HtmlDocument::getWebAssetManager()

**Comment by sonvnn** [comment]
You should update your Joomla first. Please make sure your Joomla is latest version. If you can't update your Joomla site. Pls send me your admin acc via sonlv@templaza.com. I will help you check it.

**Reply by z4pe** [reply]
I cant update, because of the Astroid package. Update information unavailable, I want to update my site on local wamp server, so you cant enter to my site.
